data/tlg002/tlg0559.tlg002.xml

```diff
@@ -0,0 +1,1942 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader>
+<fileDesc>
+<titleStmt>
+ <title xml:lang="lat">Heronis Alexandrini opera quae supersunt omnia, vol. 1: Pneumatica et automata</title>
+<author>Hero of Alexandria</author>	
+ <editor>Wilhelm Schmidt</editor>
+<sponsor>Harvard College Library</sponsor>
+ <funder>Harvard   Library   Arcadia   Fund</funder>
+<principal>Gregory Crane</principal>
+<meeting/>
+<respStmt>
+ <persName xml:id="DDD">Digital Divide Data</persName>
+ <resp>Corrected and encoded the text</resp>
+</respStmt>
+<respStmt>
+ <persName>Rhea Lesage</persName>
+ <resp>Librarian for Hellenic Studies and Coordinator for the Classics
+Collection Development</resp>
+</respStmt>
+<respStmt>
+ <persName>Bruce Robertson</persName>
+ <resp>Technical Advisor</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+ <idno type="filename">tlg0559.tlg002</idno>
+<availability>
+ <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+</availability>
+<date>2017</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United State</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="lat">Heronis Alexandrini opera quae supersunt omnia, vol. 1: Pneumatica et automata</title>
+  <editor>
+	<persName>
+	 <name xml:lang="deu">Wilhelm Schmidt</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0559.tlg001.opp-ger1">Hero of Alexandria</author>
+  <imprint>
+   <publisher>Teubner</publisher>
+   <pubPlace>Leipzig</pubPlace>
+	<date>1899</date>
+  </imprint>
+  <biblScope unit="volume">1</biblScope>
+ </monogr>
+  <ref target="https://archive.org/details/heronsvonalexandhero">The Internet Archive</ref> 
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
```

Comment:
You used the xml:lang="deu" above, so you need to define it here. Add `<language ident="deu">German</language>`.